packages/jest-cli/src/__tests__/__snapshots__/watch.test.js.snap

```diff
@@ -21,6 +21,21 @@ Watch Usage
 ]
 `;
 
+exports[`Watch mode flows allows WatchPlugins to be configured 1`] = `
+Array [
+  "
+Watch Usage
+ › Press a to run all tests.
+ › Press f to run only failed tests.
+ › Press p to filter by a filename regex pattern.
+ › Press t to filter by a test name regex pattern.
+ › Press q to quit watch mode.
+ › Press k to configured prompt.
```

Comment:
reads weird, but doesn't matter 🙂

Reply:
Good point, let me change it to `Press k to filter with a custom prompt`